Mathlib/Order/CompleteBooleanAlgebra.lean

```diff
@@ -20,7 +20,7 @@ distributive Boolean algebras.
 
 ## Typeclasses
 
-* `Order.Frame`: Frame: A complete lattice whose `⊓` distributes over `⨆`.
+* `Order.Frame`: Frm: A complete lattice whose `⊓` distributes over `⨆`.
```

Comment:
Actually, this was a mistake in the original mathlib3 PR. Just undo this change here.

Reply:
In fact, all these changes were mistakes! You can just revert them, so all this PR does is bumps the SHAs. If you'd like to fix mathlib3 as well that would be nice, but not essential.

Reply:
Will leanprover-community/mathlib#18690 do the job over there?